src/views/publish/cube-preview.ejs

```diff
@@ -0,0 +1,291 @@
+<%- include("../partials/header", t); %>
+
+<div class="govuk-width-container app-width-container">
+    <main class="govuk-main-wrapper" id="main-content" role="main">
+
+        <%- include("../partials/error-handler"); %>
+
+        <div class="govuk-panel govuk-panel--confirmation">
+            <p class="govuk-panel__title-m">
+                <%= t('publish.cube_preview.panel') %>
+            </p>
+        </div>
+
+        <h2 class="govuk-heading-l">
+            <%= t('dataset_view.key_information') %>
+        </h2>
+
+        <%
+            const publishDate = new Date(Math.max.apply(null, dataset.revisions.map((rev) => new Date(Date.parse(rev.publish_at? rev.publish_at : rev.created_at)))));
```

Comment:
Would be better to extract the code inside `new Date(...)` to a function and name it descriptively so it's self-documenting, and also just pass a publish date through to the template rather than doing the work here.